code/__DEFINES/traits.dm

```diff
@@ -526,6 +526,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// Visible on t-ray scanners if the atom/var/level == 1
 #define TRAIT_T_RAY_VISIBLE "t-ray-visible"
 #define TRAIT_FOOD_GRILLED "food_grilled"
+/// Is this food item spawned from a silver slime reaction? Prevent it from exporting for profit from cargo and make it taste disgusting.
+#define TRAIT_SILVER_SLIME_REACTION "silver_slime_reaction"
```

Comment:
Do we want to do this instead of just have a signal, what if edible component is given to a non-food item, and want to make use of the flag we're now removing? Like organs, clothes (for moths), meat materials?

Reply:
done did it like you suggested